api/users/serializers.py

```diff
@@ -2,7 +2,6 @@
 
 from api.base.serializers import JSONAPISerializer, LinksField, Link
 
-
```

Comment:
I don't believe these newline shoudl have been removed - classes should be surrounded by two blank lines.